script.js

```diff
@@ -210,9 +210,14 @@ function search(searchString, section = 'all') {
         items.forEach(item => {
             const itemName = item.getAttribute('data-name') || '';
             const itemText = item.textContent || '';
```

Comment:
this should be just item.text. And we should also search item.bottomtext

Reply:
const itemText = item.textContent + " " + item.bottomtext

Reply:
DONE
YERP